pkg/daemon/daemon.go

```diff
@@ -142,6 +144,29 @@ func NewUnikDaemon(config config.DaemonConfig) (*UnikDaemon, error) {
 		break
 	}
 
+	for _, openstackConfig := range config.Providers.Openstack {
+		openstack.MergeConfWithEnv(&openstackConfig)
+
+		// Mask password prior logging to console.
+		orig_pass := openstackConfig.Password
+		openstackConfig.Password = "<password>"
+		logrus.Infof("Bootstrapping provider %s with config %v", openstack_provider, openstackConfig)
+		openstackConfig.Password = orig_pass
+
+		p, err := openstack.NewOpenstackProvider(openstackConfig)
+		if err != nil {
+			return nil, errors.New("initializing openstack provider", err)
+		}
+		s, err := state.BasicStateFromFile(openstack.OpenstackStateFile())
+		if err != nil {
+			logrus.WithError(err).Warnf("failed to read openstack state file at %s, creating blank openstack state", openstack.OpenstackStateFile())
+			s = state.NewBasicState(openstack.OpenstackStateFile())
+		}
```

Comment:
In case the statefile is somehow corrupted, this creates a new one. Will UniK try to sync local environment and OpenStack with the new state or will it consider conpletely fresh?

It's done by all providers in the same way, so this is mainly out of curiosity.